spec/grammar.dd

```diff
@@ -1732,6 +1744,19 @@ $(GNAME DebugSpecification):
 )
 
 $(GRAMMAR
+$(GNAME StaticForeach):
+    $(D static) $(GLINK2 statement, AggregateForeach)
+    $(D static) $(GLINK2 statement, RangeForeach)
+
+$(GNAME StaticForeachDeclaration):
+    $(GLINK StaticForeach) $(GLINK2 attribute, DeclarationBlock)
+    $(GLINK StaticForeach) $(D :) $(GLINK2 module, DeclDefs)$(OPT)
```

Comment:
Can you give an example for this grammar rule? Since `static if` doesn't support trailing colon, I'm wondering if `static foreach` should not support it too.

Reply:
`static if` supports trailing semicolon.

Reply:
Hmm, I missed that even though I searched for it, nevermind then.